worldwind/src/commonMain/kotlin/earth/worldwind/layer/RenderableLayer.kt

```diff
@@ -93,5 +116,15 @@ open class RenderableLayer @JvmOverloads constructor(displayName: String? = null
                 // Keep going. Draw the remaining renderables.
             }
         }
+        try {
```

Comment:
Should we try catch inside the loop or outside here?

Reply:
done